vllm/engine/arg_utils.py

```diff
@@ -732,6 +735,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             **structured_outputs_kwargs["reasoning_parser_plugin"],
         )
 
+        # Profiler arguments
+        parser.add_argument_group(
+            title="ProfilerConfig",
+            description=ProfilerConfig.__doc__,
+        )
+
```

Comment:
I'd remove this to avoid confusion since we're not adding individual profiler arguments to the CLI outside of `--profiler-config`

```suggestion
```